tests/unit/artifact/test_koji.py

```diff
@@ -23,7 +25,9 @@ def test_koji_invalid_nvr(root_logger):
 
     provider = KojiArtifactProvider("koji.nvr:nonexistent-1.0-1.fc43", root_logger)
 
-    with pytest.raises(GeneralError, match="No build found for NVR 'nonexistent-1.0-1.fc43'."):
+    with pytest.raises(
+        GeneralError, match=re.escape("No build found for NVR 'nonexistent-1.0-1.fc43'.")
+    ):
```

Comment:
```suggestion
    with pytest.raises(GeneralError, match=r"No build found for NVR 'nonexistent-1\.0-1\.fc43'\."):
```
Prefer the raw string and escapping.

Reply:
Refactored [9ed7722](https://github.com/teemtee/tmt/pull/4119/commits/9ed77227ce3d2485d23f08658fcd89449af13483)